arrabiata/src/columns.rs

```diff
@@ -0,0 +1,82 @@
+use kimchi::circuits::expr::{CacheId, ConstantExpr, Expr, FormattedOutput};
+use std::collections::HashMap;
+use strum_macros::{EnumCount as EnumCountMacro, EnumIter};
+
+/// This enum represents the different gadgets that can be used in the circuit.
+/// The selectors are defined at setup time, can take only the values `0` or
```

Comment:
Don't we want selectors to be other values than 0/1, eg. poseidon related constants

Reply:
Selectors and public values are handled differently.

Reply:
Note: should we consider Poseidon constants as "selectors"?